### PR TITLE
Adding Pieter's BIP as 103, which it's been referred to for ages.

### PR DESCRIPTION
**Pull request by jwilkins**

### CODE REVIEW

File: bip-0103/blocksize.mediawiki [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<pre>
+  BIP: ??
```
Comment (luke-jr) [inline bip-0103/blocksize.mediawiki]: This should be fixed.

**Comment by MarcoFalke** [comment]
@sipa Is this OPL or public domain?

**Comment by btcdrak** [comment]
I think you need to amend the empty commit messages.

Also the filename needs to be `bip-0103.mediawiki`

**Comment by luke-jr** [comment]
Superceded by #238

**Pull request closed without merging**